server/src/main/java/io/deephaven/server/runner/DeephavenApiServerModule.java

```diff
@@ -154,14 +149,14 @@ public static UpdateGraphProcessor provideUpdateGraphProcessor() {
 
     private static class ThreadFactory extends NamingThreadFactory {
         public ThreadFactory(final String name) {
-            super(DeephavenApiServer.class, name, true);
+            super(DeephavenApiServer.class, name);
         }
 
         @Override
         public Thread newThread(final @NotNull Runnable r) {
             return super.newThread(() -> {
                 MultiChunkPool.enableDedicatedPoolForThisThread();
-                r.run();
+                ThreadInitializationFactory.wrapRunnable(r).run();
```

Comment:
Another case where we've opted into a different pattern for order of initializers. I think I like the one you use more.